### DISCUSSION TITLE
Problems getting the GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by Hashem-Soud1** [community/community#52515] [GitHub Education]
I am a student at the university, but I am facing problems in obtaining this package, for example, 

1-there is no button to upload images

2-You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

3-The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.

I hope that one of the officials will contact me to solve the problem, or that I will provide him with the evidence personally ... Thank you everyone

**Comment by samus-aran** [comment]
Hi @Hashem-Soud1 👋 

Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Reply by Hashem-Soud1** [reply]
Thank you @samus-aran 
I hope you can help me as soon as possible because I need that package .
I have all the legal proofs that I am a student at the university

**Comment by opegamanalitarok** [comment]
It seems like you are facing issues related to GitHub's Student Developer Pack, which offers various benefits and resources for students.

1. To upload images, you can use an external image hosting service like Imgur or upload your images to your repository using git commands or GitHub's web interface.

2. If you are a distance learner or studying remotely, you may need to provide documentation from your school that states this information to verify your eligibility for the Student Developer Pack.

3. If your school does not have a campus location in your country, you may need to provide documentation from your school that states that you are a virtual learner or studying remotely.

To resolve your issue, you can contact GitHub's support team and provide them with the necessary documentation or information to verify your eligibility for the Student Developer Pack. You can also check the eligibility requirements on the GitHub Education website to ensure that you meet all the criteria.

Hope this helps! 🙌🙌
